recipes/biogeme-optimization/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "biogeme-optimization" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biogeme_optimization-{{ version }}.tar.gz
+  sha256: a4e964d034208eb762217556112369d9f8f7dfeabe603a2dde360bca50031fca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```

Comment:
See the new lint

Reply:
Can you give a pointer for how to solve this?

Reply:
I think you add `setuptools` to the host dependencies. It says:

" If your recipe has built with only pip in the host section in the past, you likely should add setuptools to the host section of your recipe."